.stubs/pywintypes/__init__.pyi

```diff
@@ -0,0 +1,14 @@
+# SPDX-License-Identifier: MIT
+# Author: movrsi
+from typing import Final, Self, Type
+
+__all__ = ['PyHANDLE']
```

Comment:
I declare `__all__` with a tuple not a list. `__all__ = ('PyHANDLE',)`

Reply:
Heh, i never knew it could be a tuple; always assumed it was always a list due to how many projects declare it as such. Fixed.